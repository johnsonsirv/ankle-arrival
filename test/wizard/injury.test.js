/* eslint-disable react/jsx-props-no-spreading */
/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import InjuryPage from '../../src/wizard/injury';

function setup() {
  const props = {
    name: 'injuries',
    onChange: jest.fn(),
    injuries: [
      { id: 1, name: 'abdominal', code: 1 },
      { id: 2, name: 'bone', code: 2 },
    ],
  };

  const enzymeWrapper = shallow(<InjuryPage {...props} />);
  return { props, enzymeWrapper };
}

describe('<InjuryPage /> rendering', () => {
  const { enzymeWrapper } = setup();
  it('renders correctly', () => {
    expect(enzymeWrapper).toMatchSnapshot();
  });
});
