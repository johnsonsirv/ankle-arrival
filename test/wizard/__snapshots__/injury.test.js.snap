// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InjuryPage /> rendering renders correctly 1`] = `ShallowWrapper {}`;
